react/Autosuggest/Autosuggest.less

```diff
@@ -39,19 +39,15 @@
 }
 
 .suggestionsContainer_withLabel {
-  @labelRowOffset: 4;
-  top: @grid-row-height * (@suggestionContainerRowsOffset + @labelRowOffset);
-  @media @desktop {
-    @labelRowOffset: 3;
-    top: @grid-row-height * (@suggestionContainerRowsOffset + @labelRowOffset);
-  }
+  @labelRowOffset: 6;
```

Comment:
I feel like this works better as `5`. The spacing between the field (when the field has a label) and autosuggest list. At the moment it gives 2 rows, but i prefer 1. Thoughts? (Also the label is 5 rows high now that we have reduced standard copy to only have one spacing row below)

Reply:
Yeah I agree, I think 5 is better

Reply:
I'm out today, so feel free to update the PR if you like 👌

Reply:
Done